static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-post-election-copy-test.js

```diff
@@ -95,7 +93,18 @@ define([
         var contributeUrl = 'https://contribute.theguardian.com/?';
 
         var messages  = {
-            control:  '…we have a small favour to ask. More people are reading the Guardian than ever but far fewer are paying for it. And advertising revenues across the media are falling fast. So you can see why we need to ask for your help. The Guardian’s independent, investigative journalism takes a lot of time, money and hard work to produce. But we do it because we believe our perspective matters – because it might well be your perspective, too.'
+            control: {
+                title: 'Since you\'re here ...',
```

Comment:
There's the usual discrepancy in apostrophe and single quote usage - watch out 😄